src/components/SpruceForm/FieldTemplates/ArrayFieldTemplates/index.tsx

```diff
@@ -6,7 +6,7 @@ import { Body } from "@leafygreen-ui/typography";
 import { ArrayFieldTemplateProps } from "@rjsf/core";
 import { PlusButton } from "components/Buttons";
 import Icon from "components/Icon";
-import { formComponentSpacingCSS } from "components/SpruceForm/Container";
+import { formComponentSpacingCSS } from "components/SettingsCard";
```

Comment:
This makes me think this style should live somewhere else. We do have `components/styles`

Reply:
I think it makes sense colocated with the card component. It's specifically defining the card spacing, we just want to match that exact spacing for a placeholder that is used in lieu of a card.